docs/src/pages/components/data-grid/virtualization/ColumnVirtualizationGrid.tsx

```diff
@@ -45,11 +45,11 @@ function useData(rowLength: number, columnLength: number) {
 }
 
 export default function ColumnVirtualizationGrid() {
-  const data = useData(50, 1000);
+  const data = useData(500, 1000);
```

Comment:
This section is about column virtualization why not keep it very focused?
```suggestion
  const data = useData(50, 1000);
```

Reply:
There is only 1 demo in this page so I think it's nicer to see a demo of the full virtualization.

Reply:
How about we add a second demo and increase a bit this one? 

```suggestion
  const data = useData(100, 1000);
```